### PR TITLE
adjust css styles to icon and modal

### PR DESCRIPTION
**Pull request by liviaw**
add styles and give variable names

### CODE REVIEW

File: src/Components/AddMediaIcon/AddMediaIcon.module.css [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+	max-width: min(9vw, 9vh);
+	height: min(5vw, 5vh);
+	max-height: min(9vw, 9vh);
+	display: inline-block;
+	margin: 15px auto 0;
+}
+
+.addfileButton:hover {
+	border: 1px solid grey;
+	background-color: lightgrey;
```
Comment (juliamitchelmore) [inline src/Components/AddMediaIcon/AddMediaIcon.module.css]: this should also be a variable

**Review by juliamitchelmore — APPROVED** [review]
LGTM

In the Canva codebase, we tend to prefer using HEX colour values, but they don't play so nicely with alpha values unless you have adjustment function, which I don't think CSS Modules gives you. This is fine, just an FYI

Also, remember our discussion around `vw` and `vh`. Leave it for now, we can go over this sometime.

File: src/Components/AddMediaIcon/AddMediaIcon.module.css (hunk 1 above)
Comment (liviaw) [reply src/Components/AddMediaIcon/AddMediaIcon.module.css]: default values from css like white, black grey should also be a variable?

Comment (juliamitchelmore) [reply src/Components/AddMediaIcon/AddMediaIcon.module.css]: It's for consistency and reusability. 
For instance, if Jess comes along and says 'hmm, that grey should be lighter', you can change one variable to the new colour, knowing everything will be updated.

**Comment by liviaw** [comment]
on vw, vh and hex colors, will modify css once user stories are complete or after meeting with Jess F

**Comment by liviaw** [comment]
The new changes include:
- All rgb colors change to hex format
- create variables inside App.css for all colors
- add css styles for image wrapper
- add css styles for video modal using variables from App.css

**Pull request merged**